react/components/Collapsible/index.js

```diff
@@ -28,11 +28,22 @@ class Collapsible extends Component {
   constructor(props) {
     super(props)
     this.childrenRef = React.createRef()
+    this.openTimeout = null
+    this.closeTimeout = null
     this.state = {
       height: 0,
     }
   }
 
+  resetHeight = () => {
+    clearTimeout(this.timeout)
+    this.timeout = setTimeout(() => {
```

Comment:
Instead of using `setTimeout` with the timer of the transition you could use the callback of the element `onTransitionEnd`.